test/dertwothumber/component/repo_test.clj

```diff
@@ -28,4 +30,14 @@
     (testing "creating a repo"
       (repo/create-repo repo {:full-name "tobowers/test-repo" :url "http://something"}))
     (testing "fetching a repo"
-      (is (= "http://something" (:url (repo/get-repo repo "tobowers/test-repo")))))))
+      (is (= "http://something" (:url (repo/get-repo repo "tobowers/test-repo")))))
+    (testing "creating repo hook"
+      (mocking [tentacles.repos/create-hook]
```

Comment:
Awesome. Even when you break it into component, you have to test the component at some point. Now I understand what you're getting at.